Dockerfile

```diff
@@ -143,17 +125,12 @@ RUN pip install --no-cache-dir --upgrade --ignore-installed pip setuptools && \
     pip install --no-cache-dir 'scikit-image==0.15.0' && \
```

Comment:
We can move this to setup.py, looks like it already exists there, so let's remove. Please check the correct version.

Reply:
Removed in latest commit